deploy/rbd/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -55,6 +55,9 @@ rules:
   - apiGroups: [""]
     resources: ["configmaps"]
     verbs: ["get"]
+  - apiGroups: [""]
+    resources: ["serviceaccounts"]
+    verbs: ["get"]
 ---
 kind: ClusterRoleBinding
```

Comment:
@nixpanic do we need rbac for both provisioner and nodeplugin ? :thinking:

Reply:
yes, both need to connect to the KMS

Reply:
the design doc covers the `createVolume` flow (ie provisioner) part here https://github.com/ceph/ceph-csi/pull/2227/files#diff-43620beda90c9977d037956da8ade411e16b43a29efed8b37ca46c0b78834b61R79 , but the node part has not covered over there, do you think we can enhance and document `plugin` part as well in the doc ?